keyboards/bandominedoni/info.json

```diff
@@ -3,20 +3,113 @@
     "manufacturer": "3araht",
     "url": "https://github.com/3araht",
     "maintainer": "3araht",
+    "features": {
+        "midi": true,
+        "extrakey": true,
+        "bootmagic": false,
+        "command": false,
+        "console": false,
+        "encoder_map": true,
```

Comment:
`encoder_map` is never allowed to be enabled at the keyboard level.

All `rules.mk` changes regarding `encoder_map` will also need to be reverted.
```suggestion
```